src/uct/tcp/tcp.h

```diff
@@ -61,6 +65,15 @@
 #define UCT_TCP_CM_CONN_SN_MAX               UINT64_MAX
 
 
+/* TCP keepalive macro */
+#if HAVE_DECL_IPPROTO_TCP && HAVE_DECL_IPPROTO_TCP && HAVE_DECL_IPPROTO_TCP && \
```

Comment:
maybe better define a single constant in `m4`?

Reply:
such definitions in ```.m4``` is much more complicated

Reply:
ok, moved to configure script